Index: infinality/conf.src/generate60/60-group-non-tt-fonts.conf.awk
===================================================================
--- infinality/conf.src/generate60/60-group-non-tt-fonts.conf.awk.orig	2013-01-05 02:11:58.000000000 +0100
+++ infinality/conf.src/generate60/60-group-non-tt-fonts.conf.awk	2014-02-14 09:44:34.518450937 +0100
@@ -22,23 +22,6 @@
 }
 
 END {
-	printf "\t<match target=\"font\">\n"
-	printf "\t\t<test name=\"font_type\">\n"
-	printf "\t\t\t<string>NON TT Instructed Font</string>\n"
-	printf "\t\t</test>\n"
-	printf "\t\t<edit name=\"autohint\" mode=\"assign\">\n"
-	printf "\t\t\t<bool>true</bool>\n"
-	printf "\t\t</edit>\n"
-	printf "\t\t<edit name=\"hintstyle\" mode=\"assign\">\n"
-	printf "\t\t\t<const>hintslight</const>\n"
-	printf "\t\t</edit>\n"
-	printf "\t\t<edit name=\"hinting\" mode=\"assign\">\n"
-	printf "\t\t\t<bool>true</bool>\n"
-	printf "\t\t</edit>\n"
-	printf "\t\t<edit name=\"antialias\" mode=\"assign\">\n"
-	printf "\t\t\t<bool>true</bool>\n"
-	printf "\t\t</edit>\n"
-	printf "\t</match>\n\n"
 	printf "</fontconfig>\n\n"
 }
 
Index: infinality/conf.src/generate60/60-group-tt-fonts.conf.awk
===================================================================
--- infinality/conf.src/generate60/60-group-tt-fonts.conf.awk.orig	2013-01-05 02:11:58.000000000 +0100
+++ infinality/conf.src/generate60/60-group-tt-fonts.conf.awk	2014-02-14 09:45:47.492311657 +0100
@@ -9,9 +9,6 @@
 
 {
 	printf "\t<match target=\"font\">\n"
-	printf "\t\t<test name=\"force_autohint\">\n"
-	printf "\t\t\t<bool>false</bool>\n"
-	printf "\t\t</test>\n"
 	printf "\t\t<test name=\"family\">\n"
 	printf "\t\t\t<string>" 
 	printf $0
